src/rviz/TSRMarker.cpp

```diff
@@ -0,0 +1,14 @@
+#include <aikido/rviz/TSRMarker.hpp>
+
+using dart::dynamics::SimpleFrame;
+
+namespace aikido {
+namespace rviz {
+
+TSRMarker::TSRMarker(std::vector<std::unique_ptr<SimpleFrame>> tsrFrames)
```

Comment:
> I don't feel particularly strongly either way, so let's leave it up to @gilwoolee to decide.

:+1: 

My only concern is that the caller is _not able to pass lvalue anyway_ (whether or not the argument is rvalue reference because copy assignment of `std::unique_ptr` is not defined), but explicit rvalue reference argument could give us hint why the caller cannot pass lvalue.

Alright, let's pass the baton to @gilwoolee. :smiling_imp: